app/models/alchemy/page_version.rb

```diff
@@ -46,7 +46,7 @@ def still_public_for?(time = Time.current)
     end
 
     def element_repository
-      ElementsRepository.new(elements.includes({ contents: :essence }, :tags))
+      ElementsRepository.new(elements)
```

Comment:
How do we make sure that people eager load properly if not done by our controller? Let's move this into the extracted PR and discuss this further.